libimage/image.go

```diff
@@ -112,6 +112,11 @@ func (i *Image) Names() []string {
 	return i.storageImage.Names
 }
 
+// setNames sets the names of the image.
```

Comment:
```suggestion
// setNames sets the names of the image.
//
// WARNING: this only affects the in-memory values, they are not written into the backing storage.
```
… and the function name should also reflect this. `setEphemeralNames`??? Or maybe the data should be returned by `ListImages` in some other way, I don’t feel that strongly about that part.